### DISCUSSION TITLE
SMTP Provider Extension: SendGrid

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#7852] [Feature Requests]
Hi,

Could you consider to add Sendgrid as email replay provider like MailGun?
Thanks.

**Comment by rijkvanzanten** [comment]
Sendgrid supports SMTP as well, so unless there's a community contribution through a PR, there's no plan to add any native Sendgrid integrations as you can already use Directus with Sendgrid as is.

**Reply by naskio** [reply]
@rijkvanzanten I am willing to make a PR to add native support for SendGrid (over HTTP)
Do I need to create a new Issue or make a PR directly (since the discussion is already ongoing) ?
Here is draft PRs ([code](https://github.com/directus/directus/pull/15384)+[docs](https://github.com/directus/docs/pull/122))

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
